text_mining.py

```diff
@@ -0,0 +1,82 @@
+def tweet_list(hashtag, number):
+    """
+    Takes in a string and value find and returns a list of tweets the value
+    long that contain the string
+    """
+    from pattern.web    import Twitter
+
+    t = Twitter()
+    i = None
+    tweets = []
+    for j in range(number):
+        for tweet in t.search(hashtag, start=i, count=1):
+            # print tweet.text
+            # print
+            tweets.append(tweet.text)
+            i = tweet.id
+    return tweets
+
+def hastag_setiment(hashtag):
```

Comment:
"hastag_setiment"?
As long as functions have consistent names, it's not super important what they are--but, this seems like it miiiight be a typo?